src/modules/RA/BaseLayer/components/ZoomInput/index.js

```diff
@@ -0,0 +1,3 @@
+import ZoomInput from './ZoomInput';
```

Comment:
Why do you create a "Directory" ?

Reply:
A habit in the case we need to use HOC with the component :
1/ I can keep the component clean without theses "decorators"
2/ I don't have to transform the file into a folder with to files when adding the first HOC

Reply:
Why not, but:
- I don't feel this component will get a HOC one day and if for each component we create this structure it will be unreadable very quickly. I think we should use this logic only for component where HOC is probable ("glues" component for example)
- In functional component word hooks are preferred over HOC (try to make a functional HOC :D) so we can assume we will use a hooks and not a HOC lateron.

Reply:
Fixed in one file